src/config.ts

```diff
@@ -0,0 +1,12 @@
+import { cosmiconfig } from 'cosmiconfig';
+import JSON5 from 'json5';
+
+const explorer = cosmiconfig('astroturf', {
+  searchPlaces: ['package.json', `.astroturfrc`, `.astroturfrc.json`],
+  loaders: {
+    json: (_file: string, content: string) => JSON5.parse(content),
+    noExt: (_file: string, content: string) => JSON5.parse(content),
```

Comment:
I am intentionally limiting this Not JS initially, we can add support later if needed but it seems like going more minimal now is a good idea. Also screw yaml

Reply:
lol